README.md

```diff
@@ -22,97 +22,100 @@ A collection of all things awesome related to the [Dagster](https://dagster.io)
+- [Datadex](https://github.com/datonic/datadex)
+- [Catalyst Cooperative PUDL](https://github.com/catalyst-cooperative/pudl)
+- [MIT Open Learning](https://github.com/mitodl/ol-data-platform/tree/main/src/ol_orchestrate)
+- [Dagster Open Platform](https://github.com/dagster-io/dagster-open-platform)
+- [Dagster Graph Project](https://github.com/sephib/dagster-graph-project)
+- [Portable Data Stack](https://github.com/cnstlungu/portable-data-stack-dagster)
+- [Practical Data Engineering Project](https://github.com/sspaeti-com/practical-data-engineering)
+- [Coches.net Dashboard](https://github.com/franloza/coches-net-dashboard)
 
 ## Discussion
 
 ### Community
 
-- [Dagster Slack](https://dagster.io/slack) - Join the community and Dagster Labs team members on Slack
 - [Dagster Github Discussion](https://github.com/dagster-io/dagster/discussions) - Discuss features and contributions on GitHub
+- [Dagster Slack](https://dagster.io/slack) - Join the community and Dagster Labs team members on Slack
 
 ## Company Projects
 
-- Bestplace - [Building a Modern Data Analytics Platform with Dagster](https://youtu.be/9WKtBFg2bUo)
 - Doordash - [Increasing Operational Efficiency with Scalable Forecasting](https://doordash.engineering/2021/08/31/increasing-operational-efficiency-with-scalable-forecasting/)
-- Drizly - [Building shared spaces for data teams at Drizly](https://dagster.io/blog/shared-spaces-at-drizly)
-- Geomagical Labs (Ikea) - [Building Customer-Facing Applications with Dagster](https://youtu.be/1buRROJ3m2s)
-- GoodEggs - [Good Data at Good Eggs](https://dagster.io/blog/good-eggs-0)
-- LINE (Thailand) - [Build Data Application with Dagster](https://www.youtube.com/watch?v=Te-7u5Xgheo)
+- VMware - [Monitoring Driven Development with Dagster](https://youtu.be/fYJBN6MAtbE?t=368)
 - MIT Open Learning - [Running Dagster in Production](https://youtu.be/5wS7BpjtRlI)
+- Discord - [How Discord Uses Open-Source Tools for Scalable Data Orchestration & Transformation](https://discord.com/blog/how-discord-uses-open-source-tools-for-scalable-data-orchestration-transformation)
 - Mapbox - [Incrementally Adopting Dagster at Mapbox](https://dagster.io/blog/incremental-adoption-mapbox)
 - Prezi - [Migrating Data Pipelines to Dagster](https://youtu.be/-zv-wvbJXUM)
-- VMware - [Monitoring Driven Development with Dagster](https://youtu.be/fYJBN6MAtbE?t=368)
+- LINE (Thailand) - [Build Data Application with Dagster](https://www.youtube.com/watch?v=Te-7u5Xgheo)
+- Telekom (T Magenta) - [Scaling Data @Telekom](https://www.youtube.com/watch?v=GX508If6ol0&t=1s)
+- Geomagical Labs (Ikea) - [Building Customer-Facing Applications with Dagster](https://youtu.be/1buRROJ3m2s)
+- GoodEggs - [Good Data at Good Eggs](https://dagster.io/blog/good-eggs-0)
+- Drizly - [Building shared spaces for data teams at Drizly](https://dagster.io/blog/shared-spaces-at-drizly)
+- Bestplace - [Building a Modern Data Analytics Platform with Dagster](https://youtu.be/9WKtBFg2bUo)
```

Comment:
Adding videos/articles from Discord and T Magenta

Reply:
nice!